packages/cf-wallet.js/src/app-instance.ts

```diff
@@ -17,11 +17,28 @@ export class AppInstance {
    */
   readonly id: AppInstanceID;
 
+  // Application-specific fields
   readonly appDefinition: Address;
   readonly abiEncodings: AppABIEncodings;
+  readonly timeout: BigNumber;
+
+  // Funding-related fields
   readonly myDeposit: BigNumber;
   readonly peerDeposit: BigNumber;
-  readonly timeout: BigNumber;
+
+  readonly twoPartyOutcomeInterpreterParams?: {
```

Comment:
This seems like it's repeated multiple times. Maybe define it in the `types` package?

(Same with `ethTransferInterpreterParams`)

Reply:
I was hesitant to do this because I want to remove this code in a future PR so it is a bit more general. Right now it is explicitly ugly because this _is_ an ugly field to add to this object.

Reply:
I share the hesitation, just think keeping things consistent even in the interim is generally good until this future PR does happen which can refactor things more generally then.

Reply:
Added types.